src/index.js

```diff
@@ -8,6 +8,7 @@ import Home from "./pages/home";
 import Login from "./pages/login";
 import Question from "./pages/question";
 import Friend from "./pages/friend";
+import Previous from "./pages/previous";
```

Comment:
Previous what? Is this previous questions? If so, let's name it that, so it's clearer.